gpbft/options.go

```diff
@@ -117,33 +118,35 @@ func WithMaxCachedMessagesPerInstance(v int) Option {
 	}
 }
 
-var defaultRebroadcastAfter = exponentialBackoffer(1.3, 3*time.Second, 30*time.Second)
+var defaultRebroadcastAfter = exponentialBackoffer(1.3, 0.1, 3*time.Second, 30*time.Second)
 
 // WithRebroadcastBackoff sets the duration after the gPBFT timeout has elapsed, at
 // which all messages in the current round are rebroadcast if the round cannot be
 // terminated, i.e. a strong quorum of senders has not yet been achieved.
 //
-// The backoff duration grows exponentially up to the configured max. Defaults to
-// exponent of 1.3 with 3s base backoff growing to a maximum of 30s.
-func WithRebroadcastBackoff(exponent float64, base, max time.Duration) Option {
+// The backoff duration grows exponentially up to the configured max and is randomized by "spread"
+// (a number between 0 and 1). Defaults to exponent of 1.3 with 3s base backoff growing to a maximum
+// of 30s, randomized by 10% (5% in either direction),
+func WithRebroadcastBackoff(exponent, spread float64, base, max time.Duration) Option {
```

Comment:
Thinking out loud (and maybe a bit of a language question since I am not a native speaker): would jitter be a better word compared to spread?

Reply:
Jitter is also correct. It was actually my first thought but I picked spread to make it clear that it's intentional.

But if jitter is also your first thought... it sounds like the word to use.